masonry/src/widget/widget_pod.rs

```diff
@@ -504,26 +505,12 @@ impl<W: Widget> WidgetPod<W> {
     ///
     /// [`layout`]: Widget::layout
     pub fn layout(&mut self, parent_ctx: &mut LayoutCtx, bc: &BoxConstraints) -> Size {
```

Comment:
Why is this still here?

Reply:
To keep the diff small. We can change all the widgets in a follow-up PR. I'll include a comment to that effect.

Reply:
Done.

Reply:
All of the widgets already appear to be translated in this PR?